### PR TITLE
Fields of Interest dropdown selection | sign-up

### PR DESCRIPTION
**Pull request by aktharrr**
Fixes https://github.com/AIESEC-LK/dev.team/issues/40

### CODE REVIEW

File: src/app/interfaces/company.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export interface Company {
+  company_name: string,
+  company_address: string,
+  company_email: string,
+  company_phone: string,
+  fulll_name: string,
```
Comment (kaneeldias) [inline src/app/interfaces/company.ts:6]:
```suggestion
  full_name: string,
```

File: functions/index.js [hunk 2]
```diff
@@ -33,9 +33,31 @@ exports.onParticipantUpdate = functions.firestore
...
+//   .document('companies/{company_name}')
+//   .onUpdate(async (snap, context) => {
+//     if (snap.before.data().lastModifiedTimestamp !== snap.after.data().lastModifiedTimestamp) return;
+//     const participantRef = db.collection('companies').doc(snap.after.id);
+//     const timestamp = new Date().toISOString();
+//     await participantRef.set({
+//       lastModifiedTimestamp: timestamp
+//     }, { merge: true });
+//   });
+
```
Comment (kaneeldias) [inline functions/index.js:60]: Shall we remove the commented out areas from the PR?

Comment (kaneeldias) [reply functions/index.js:60]: There's a few more in this PR

**Comment by aktharrr** [comment]
Commented codes removed

**Pull request merged**